templates/csharp/src/Appwrite/Models/OrderType.cs.twig

```diff
@@ -0,0 +1,8 @@
+namespace Appwrite
```

Comment:
We can use a twig var here, instead of Appwrite hard-coded, might use it for other stuff in the future